.github/workflows/awie.yml

```diff
@@ -27,8 +27,22 @@ jobs:
     with:
       version_file: centreon-awie/www/modules/centreon-awie/conf.php
 
-  veracode-analysis:
+  check-version:
     needs: [get-version]
+    runs-on: ubuntu-22.04
+    if: ${{ contains(fromJson('["testing"]'), needs.get-version.outputs.stability) && github.base_ref == 'master' }}
+
+    steps:
+      - name: Check version
+        uses: ./.github/actions/check-version
+        with:
+          module_name: awie
+          major_version: ${{ needs.get-version.outputs.major_version }}
+          minor_version: ${{ needs.get-version.outputs.minor_version }}
+          stability: ${{ needs.get-version.outputs.stability }}
+
+  veracode-analysis:
+    needs: [get-version, check-version]
```

Comment:
```suggestion
    needs: [check-version]
```

Reply:
since veracode job relies on the get-version output, linter/gh may complain about it as being a bad practice, we should leave both get-version and check-version